src/scripts/components/header.ts

```diff
@@ -0,0 +1,18 @@
+const closeBTN: HTMLElement | null = document.querySelector(
+	'.menu__button-close'
+);
+const menuSlider: HTMLElement | null = document.querySelector('.menu__slider');
+const menuBTN: HTMLElement | null =
+	document.querySelector('.header__menu-icon');
+
+menuBTN?.addEventListener('click', function () {
+	if (menuSlider) {
+		menuSlider.style.width = '100%';
```

Comment:
rather than explicitly setting the width you could instead toggle a class on `.menu` or on `.menu__slider` for example `.menu--active` or `.menu__slider--active` which sets the width instead

Reply:
Done